Applications/_Plugins/Common/Misc.cpp

```diff
@@ -19,8 +19,14 @@
 
 #include "Misc.h"
 
+#ifndef _WIN32
+#include <chrono>
+static std::chrono::time_point<std::chrono::steady_clock> misc_start = std::chrono::steady_clock::now();
```

Comment:
This may not be correct. Please double check. We just need a timer to have started so we can have a cross platform equivalent of `MillisecondsNow`